src/nomad_simulations/schema_packages/outputs.py

```diff
@@ -38,8 +37,10 @@
     XASSpectrum,
 )
 
+from .common import Time
 
-class Outputs(ArchiveSection):
+
+class Outputs(Time):
```

Comment:
Why does Outputs need to inherit from Time? Is there some situation where the simulation package is outputting such detailed timing info? 

I guess maybe you are thinking of the workflows. Is the plan to reuse the output class directly for workflows?

Reply:
Indeed the task link at least for the general simulation workflow class references the output. For geometry optimization, I do a renormalization to extract the system to as I feel it is the natural input and output. The Simulation class as I understand, will not be extended to cover workflow methods and results, right? So I still would like to put the workflow-related quantities under workflow.method and workflow.results as in the old def.